docs/tab-widgets/install-agents.asciidoc

```diff
@@ -0,0 +1,438 @@
+// tag::go[]
```

Comment:
Given relative language popularity: would it be sensible (and possible) to retain alphabetical ordering, but make the Java agent the initially active tab?

Reply:
An interesting idea! Let me play around with it.

Reply:
I like it. Done. 🙇